core/vm/contracts.go

```diff
@@ -163,12 +164,29 @@ func ActivePrecompiles(rules params.Rules) []common.Address {
 	}
 }
 
+type statefulPrecompileIntf interface {
+	RunExtra(
```

Comment:
This is a workaround and the method cannot be called Run because it conflicts with PrecompiledContract. We should replace this with something else but these inputs are used in the existing precompiles.